task-list/app/views/tasks/index.html.erb

```diff
@@ -24,26 +23,26 @@
     <tbody>
       <% @tasks.each do |task| %>
       <tr>
-        <td class="bigger_checkbox">
+        <td class="bigger_checkbox" class="bigger">
           <% if task.completed_date.nil? %>
-            ☐
+            <%= link_to "☐", "/tasks/#{task.id}/complete", method: :patch %>
             <% strike = "incomplete" %>
           <% else %>
             ☑
             <% strike = "complete" %>
           <% end %>
         </td>
-        <td class=<%= strike %>>
-          <%= link_to task.name.capitalize, "/tasks/#{task.id}" %>
+        <td class=<%= strike %> >
```

Comment:
The `class` attribute on this tag should be quoted, like: `class="<%= strike %>"`.